content/docs/tutorials/get-started/data-access.md

```diff
@@ -1,31 +1,39 @@
 # Data Access
 
-We've seen how to [version data](/doc/tutorials/get-started/data-versioning) for
-sharing among team members or environments of the same project. But what about
-reusing your data and models from an existing DVC project in other projects, or
-on a production deployment?
+Okay, now that we've learned how to _track_ data and models with DVC and how to
+version them with Git, next question is how can we _use_ these artifacts outside
+of the project? How do I download a model to deploy it? How do I download a
+specific version of a model? How do I reuse datasets across different projects?
 
-<abbr>DVC repositories</abbr> serve as an entry point for your data. The
-following CLI commands and an API are available to access any version of it,
-from any machine where DVC is installed.
+> Especially, these questions come up very fast when you see the way DVC saves
+> files to the remote storage, e.g.
+> `s3://dvc-public/remote/get-started/fb/89904ef053f04d64eafcc3d70db673` 😱
+> instead of `model.pkl` or `data.xml`.
 
-## Find a dataset
+Remember those `.dvc` files `dvc add` generates? Or `dvc.yaml` and `dvc.lock`
+pipeline files? They (and their versions in Git) contain all the information
+needed to access any version of any dataset or model in the remote storage. All
+you need is to have access to the <abbr>DVC repository</abbr>. Practically, it
+means a URL to a Git repo that you have access to.
```

Comment:
> All you need is to have access to the <abbr>DVC repository</abbr>. Practically, it means a URL to a Git repo that you have access to.

Is this true? You also need access to the remote storage. Also, "All you need" could be interpreted as condescending. Best to say something more specific like "you no longer need to keep track of access to a, b, and c... but only to the DVC repo (and remote storage if it's not public)"

Reply:
it's not exactly true, you are right ... I sacrificed a bit for simpliciy

Reply:
redid it again